scripts/tasks/task_compileenv.jl

```diff
@@ -8,8 +63,18 @@ project_filename = isfile(joinpath(env_to_precompile, "JuliaProject.toml")) ? jo
 
 project = Pkg.API.read_project(project_filename)
 
-used_packages = Symbol.(collect(keys(project.deps)))
+# Read the configuration file
+config = read_configuration(env_to_precompile)
+dev_packages = find_dev_packages(env_to_precompile)
+
+# Assemble the arguments for the `create_sysimage` function
+used_packages = filter(x -> !(x in dev_packages || x in config[:excluded_packages]), Symbol.(collect(keys(project.deps))))
+precompile_statements = config[:precompile_statements_file]
+precompile_execution = config[:precompile_execution_file]
 
-@info "Now building a custom sysimage for the environment '$env_to_precompile'."
+@info "Now building a custom sysimage for the environment '$env_to_precompile', excluding dev packages '$dev_packages'."
```

Comment:
Might be better to show a list of packages that get compiled instead?

Reply:
Indeed, the script now prints out a list of included packages.